Source/FieldSolver/MagnetostaticSolver/MagnetostaticSolver_K.H

```diff
@@ -0,0 +1,251 @@
+/* Copyright 2022 S. Eric Clark, LLNL
```

Comment:
Could this file be moved into the `ablastr/fields` folder, in order to be consistent with this similar file:
https://github.com/ECP-WarpX/WarpX/tree/development/Source/ablastr/fields
In this case, we could rename this file `VectorPoissonSolver`.